index.html

```diff
@@ -2108,24 +2108,30 @@ <h3>Agility and Hardening</h3>
 not provide the desired protections.
         </p>
         <p>
-<dfn class="lint-ignore">Cryptographic hardening</dfn> is a practice where one
+<dfn class="lint-ignore">Cryptographic layering</dfn> is a practice where one
 designs <em>rarely connected</em> information security systems to
 <em>employ multiple primitives and/or algorithms at the same time</em>. The
-primary goal of cryptographic hardening is to enable systems to survive the
+primary goal of cryptographic layering is to enable systems to survive the
 failure or one or more cryptographic algorithms or primitives without losing
 cryptographic protection on the payload. For example, digitally signing a single
 piece of information using RSA, ECDSA, and Falcon algorithms in parallel would
 provide a mechanism that could survive the failure of two of these three digital
 signature algorithms. When a particular cryptographic protection is compromised,
 such as an RSA digital signature using 768-bit keys, systems can still utilize
 the non-compromised cryptographic protections to continue to protect the
-information.
+information. Developers are urged to take advantage of this feature for all
+signed content that might need to be protected for a year or longer.
+        </p>
+        <p>
+This specification provides for both forms of agility. It provides for
```

Comment:
@msporny — `This specification provides for both forms of agility.` doesn't quite make sense when those forms are `cryptographic agility` and `cryptographic layering` (which isn't so much "agile", to my mind).  Maybe `both forms of cryptographic enhancement`?